File: C/5-simple-math.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
    char num1[8], num2[8];

    printf("What is the first number? ");
    scanf("%8s", num1);
    printf("What is the second number? ");
    scanf("%8s", num2);

    int dec1 = atoi(num1), dec2 = atoi(num2);

    printf("%d + %d = %d\n%d - %d = %d\n%d * %d = %d\n%d / %d = %d\n", dec1, dec2, dec1+dec2, dec1, dec2, dec1-dec2, dec1, dec2, dec1*dec2, dec1, dec2, dec1/dec2);
    
    return 0;
}

File: go/5-simple-math.go
package main

import "fmt"

func main() {
	var first, second int
	
	fmt.Printf("What is the first number? ")
	fmt.Scan(&first)
	fmt.Printf("What is the second number? ")
	fmt.Scan(&second)

	fmt.Printf("%d + %d = %d\n", first, second, first + second)
	fmt.Printf("%d - %d = %d\n", first, second, first - second)
	fmt.Printf("%d * %d = %d\n", first, second, first * second)
	fmt.Printf("%d / %d = %d\n", first, second, first / second)
}